request-server.go

```diff
@@ -183,10 +184,12 @@ func (rs *RequestServer) packetWorker(pktChan chan requestPacket) error {
 func cleanPath(pkt *sshFxpRealpathPacket) responsePacket {
 	path := pkt.getPath()
 	if !filepath.IsAbs(path) {
-		path = "/" + path
+		// prevent double slash (e.g. on windows, / paths are not absolute)
+		path = "/" + strings.TrimPrefix(path, "/")
 	} // all paths are absolute
 
-	cleaned_path := filepath.Clean(path)
+	cleaned_path := filepath.ToSlash(filepath.Clean(path))
```

Comment:
don't create a new variable, overwrite path so that the unclean version cannot be used by mistake.